test/libsolidity/semanticTests/events/event_emit_other_contract.sol

```diff
@@ -0,0 +1,22 @@
+contract C2 {
```

Comment:
I'd rename this test to `event_emit_from_other_contract`. Originally I thought you would be trying to emit a contract instance :P